mesh/mesh.go

```diff
@@ -369,11 +369,8 @@ func (msh *Mesh) persistLayerHashes(ctx context.Context, from, to types.LayerID)
 			return err
 		}
 
-		prevHash, err := msh.getAggregatedLayerHash(i.Sub(1))
-		if err != nil {
-			logger.With().Debug("failed to get previous aggregated hash", i, log.Err(err))
-			return err
-		}
+		prevHash := msh.GetAggregatedLayerHash(i.Sub(1))
```

Comment:
this change cause the code to ignore error. this is not safe. we need to error out if we failed to get previous layer hash instead of using EmptyLayerHash

Reply:
done